lib/bumper.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: UTF-8
```

Comment:
next you will move to 
lib/robox/bump.rb
maybe we will split in more logical names like
bump, git to be continued